src/components/LogInComponent/LoginFormComponent.jsx

```diff
@@ -1,48 +1,99 @@
-import { useSelector } from 'react-redux';
-import languageSelector from '../../assets/languages/languageSelector';
+import { useSelector } from "react-redux";
+import languageSelector from "../../assets/languages/languageSelector";
+import { useState } from "react";
+import axios from "axios";
+import Cookies from "universal-cookie";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useDispatch } from "react-redux";
+import { setUser } from "../../store/actions/user.action";
 
-const LoginFormComponent = ({ setWhichForm }) => {
+const LoginFormComponent = ({ setWhichForm, closeModal }) => {
+  const cookies = new Cookies();
+  const dispatch = useDispatch();
+  const authUrl = process.env.REACT_APP_AUTH_URL;
   const language = useSelector(state=> state.languageReducer);
+  const [dataUser, setDataUser] = useState({
+    email: "",
+    password: ""
+  });
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const { data } = await axios.post(`${authUrl}/local/login`, dataUser);
+      cookies.set("token", data.token);
+      cookies.set("name", data.data.name);
+      cookies.set("last_name", data.data.last_name);
+      cookies.set("email", data.data.email);
+      dispatch(setUser(data.data));
+      toast.success(languageSelector(language, "logInSuccessfull"));
+
+      setTimeout(() => {
+        closeModal();
+      }, 6000);
```

Comment:
I consider avoiding any lack of memory using this kind of implementation with setTimeout:
```js
      const timerModal = setTimeout(() => {
        closeModal();
      }, 6_000);
    clearTimeout(timerModal);
```

Reply:
Resolved, but i ussed a state a useeffect to get the expected behavior

Reply:
The refactor was better yet, using hooks was a relevant change, good one!